tap_googleads/tap.py

```diff
@@ -137,6 +157,8 @@ def setup_mapper(self):
         
     def discover_streams(self) -> List[Stream]:
         """Return a list of discovered streams."""
-        if self.config["enable_click_view_report_stream"]:
-            STREAM_TYPES.append(ClickViewReportStream)
-        return [stream_class(tap=self) for stream_class in STREAM_TYPES]
+        streams = [stream_class(tap=self) for stream_class in STREAM_TYPES]
+        if self.config.get("enable_click_view_report_stream"):
+            streams.append(ClickViewReportStream(tap=self))
+        streams.extend(CustomQueryStream(tap=self, custom_query=q) for q in self.config.get("custom_queries_array", []))
```

Comment:
Unnecessary default to `[]`, as this is already done [here](https://github.com/Matatika/tap-googleads/pull/93/files#diff-891bc5e28a24bcdfcd270b383af35ef4837bb64f9234b70cfcca501132b673deR147) - there is guaranteed to be a `custom_queries_array` entry at this point:

```suggestion
        streams.extend(CustomQueryStream(tap=self, custom_query=q) for q in self.config["custom_queries_array"])
```

Reply:
See above correction to setting name also:

```suggestion
        streams.extend(CustomQueryStream(tap=self, custom_query=q) for q in self.config["custom_queries"])
```